dev-tools/mage/manifest/manifest.go

```diff
@@ -68,16 +69,75 @@ func DownloadManifest(manifest string) (tools.Build, error) {
 }
 
 func resolveManifestPackage(project tools.Project, pkg string, reqPackage string, version string) []string {
+	var val tools.Package
+	var ok bool
+
+	// Try the normal/easy case first
 	packageName := fmt.Sprintf("%s-%s-%s", pkg, version, reqPackage)
-	val, ok := project.Packages[packageName]
+	val, ok = project.Packages[packageName]
```

Comment:
Is there a sample manifest we can check in here and use to write a unit test against this function? Seeing an actual example would also help me understand if this is correctly doing what it is supposed to do.

Reply:
Adding a few manifests (snapshot, staging, and ind. agent staging) to test this function.